src/blocks/big-search.css

```diff
@@ -0,0 +1,45 @@
+.big-search__input {
+  font-size: 16px;
+  margin: 0 auto;
+  width: 100%;
+  text-indent: 62px;
+  border: none;
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19;
+  background-position: 26 24;
+}
+
+.main__big-search {
+  display: flex;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin: 20px 8px 0;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.main__big-search:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.main__big-search:focus {
+  background: linear-gradient(180deg, #fff 100%, #f6f6f7 100%);
+}
+
+.big-search__input:focus {
+  outline: none;
+  border-radius: 3px;
+  box-shadow: 1px 4px 4px #00000040;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19;
```

Comment:
The 'background-size' property is missing a unit. It should be specified in a valid CSS unit like 'px', 'em', etc.